### DISCUSSION TITLE
Cyberpunk 2077 support

### DISCUSSION THREAD

**Discussion by Traderain** [WolvenKit/WolvenKit#288] [Feature Requests]
## 👋 Welcome!
  We’re using Discussions as a place to talk about the upcoming Cyberpunk 2077 game by CDPR  with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by pritster5** [comment]
Are most of the file formats used for assets in CP2077 the same as TW3?

**Comment by Mallock** [comment]
Hello, I was interested reverse engineering Cyberpunk 2077 before finding out this project. Tried different tools until I found out Ghidra a reverse engineer tool that NSA has made free to use. It seemed to do good job but the biggest problem was that it doesn't have access to PDB info to map the function calls. Has anyone else tried to reverse engineer Cyberpung with Ghidra?
